syft/workers/websocket_client.py

```diff
@@ -96,3 +94,40 @@ def _recv_msg(self, message: bin) -> bin:
                     "Websocket connection closed and creation of new connection failed."
                 )
         return response
+
+    def list_objects_remote(self):
+        message = self.create_message_execute_command(
+            command_name="list_objects", command_owner="self"
+        )
+        serialized_message = sy.serde.serialize(message)
+        # Send the message and return the deserialized response.
+
+        response = self._recv_msg(serialized_message)
+        return sy.serde.deserialize(response)
+
+    def objects_count_remote(self):
+        message = self.create_message_execute_command(
+            command_name="objects_count", command_owner="self"
+        )
+        serialized_message = sy.serde.serialize(message)
+        # Send the message and return the deserialized response.
+
+        response = self._recv_msg(serialized_message)
+        return sy.serde.deserialize(response)
+
+    def __str__(self):
+        """Returns the string representation of a Websocket worker.
+
+        A to-string method for websocket workers that includes information from the websocket server
+
+        Returns:
+            The Type and ID of the worker
+
+        """
+        out = "<"
+        out += str(type(self)).split("'")[1].split(".")[-1]
```

Comment:
This is a bit confusing, can you add a comment about what's going on on this line?